tests/cpp/test_matmul.cpp

```diff
@@ -5375,6 +5377,158 @@ TEST_F(HopperMatmulTest, EpilogueSiluPersistentBroadcastInputs) {
       at::allclose(cg_outputs[0].as<at::Tensor>(), tv11_ref, 5e-2, 1e-1));
 }
 
+TEST_F(BlackwellMatmulTest, EpilogueBiasPersistentBroadcastInputs) {
+  EnableOptionsGuard eog;
+  EnableOptionsGuard::getCurOptions().set(EnableOption::FuseMultipleMatmuls);
+
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  constexpr int64_t M = 8192, N = 8192, K = 8192;
+  const auto dtype = DataType::BFloat16;
+
+  auto tv0 = makeContigConcreteTensor({-1, 1, -1}, dtype); // M, 1, K
```

Comment:
nitpick: Do we care about the broadcast part?

Reply:
I think it is the same situation as Hopper, they are both gmem->smem->tensor core, so the input has to be already broadcated.